gaphor/i18n.py

```diff
@@ -7,11 +7,19 @@
 import functools
 import gettext as _gettext
 import importlib.resources
+import locale
 import logging
+import os
+import sys
 import xml.etree.ElementTree as etree
 
 log = logging.getLogger(__name__)
 
+if sys.platform == "win32" and os.getenv("LANG") is None:
+    language, _ = locale.getlocale()
```

Comment:
Using getlocale() here is not a good idea: https://stackoverflow.com/a/25691701

Reply:
Hey @mathiascode, this is working for me if I change the language on Windows, but glad to learn a better way. Are you suggesting we use ctypes to get the user's language on Windows?

Reply:
You can test it by setting the Windows regional format to something else than the system language.

To my knowledge, using the GetUserDefaultUILanguage API with ctypes is the most appropriate way of receiving the system language (unfortunately). It has worked well for us in Nicotine+ so far: https://github.com/nicotine-plus/nicotine-plus/blob/95204e34976c5e8d697d3e5ab9cc0e42cc3965e0/pynicotine/i18n.py#L64